07-Extract-NDVI-From-Rasters-in-R.Rmd

```diff
@@ -84,76 +87,78 @@ This lesson is a part of a series of raster data in R lessons:
 * [Lesson 05 - Raster Time Series Data in R]({{ site.baseurl}}/R/Raster-Times-Series-Data-In-R/)
 * [Lesson 06 - Plot Raster Time Series Data in R Using RasterVis and LevelPlot]({{ site.baseurl}}/R/Plot-Raster-Times-Series-Data-In-R/)
 * [Lesson 07- Extract NDVI Summary Values from a Raster Time Series]({{ site.baseurl}}/R/Extract-NDVI-From-Rasters-In-R/)
-</div>
-
-```{r load-libraries }
-
-library(raster)
-library(rgdal)
-library(ggplot2)
 
-```
+###Sources of Additional Information
 
-#About the Time Series Data
+* <a href="http://cran.r-project.org/web/packages/raster/raster.pdf" target="_blank">
+Read more about the `raster` package in R.</a>
 
-In this lesson, we are working with the same set of rasters that we used in 
-[{{ site.baseurl}} /R/Raster-Times-Series-Data-In-R/ ](Lesson 05 - Time Series Rasters in R), 
-that were derived from the Landsat satellite - in `GeoTiff` format. Each
-raster covers the <a href="http://www.neoninc.org/science-design/field-sites/harvard-forest" target="_blank">NEON Harvard Forest field site</a>.
+</div>
 
+#Why Summarize NDVI Raster Data
+While raster NDVI data is great for imaging, however, as ecologists we often
+want to plot a value for greenness on a plot or compare change in NDVI values 
+with air temperature or precipitation.  To do this we have to get summary values
+from the raster NDVI data. 
 
 ##Getting Started 
+In this lesson, we are working with the same set of rasters used in the
+[Raster Time Series Data in R ]({{ site.baseurl}} /R/Raster-Times-Series-Data-In-R/) and [Plot Raster Time Series Data in R Using RasterVis and Levelplot ]({{ site.baseurl}}/R/Plot-Raster-Times-Series-Data-In-R/)
+lessons. This data is derived from the Landsat satellite and stored in GeoTIFF
```

Comment:
The "already done in previous lessons bit"  lets people who are going through the series know that they don't need to reload the data (currently this is done across all in Spatial Raster and Time series lessons).